api-lambda-ses/example-pattern.json

```diff
@@ -0,0 +1,54 @@
+{
+  "title": "Amazon API Gateway REST API to AWS Lambda to Amazon SES",
+  "description": "This sample project demonstrates publishing an email through SES by invoking a Lambda function through a REST API.",
+  "language": "Python",
+  "level": "200",
+  "framework": "SAM",
+  "introBox": {
+    "headline": "Create an Amazon API Gateway REST API that invokes a Lambda function to send email through SES.",
+    "text": ["This pattern deploys a Amazon API Gateway REST API with Lambda Function integration and SES"]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-patterns/blob/main/apigw-lambda-ses",
```

Comment:
Update with the correct project directory

Reply:
Missed this one. Thanks for pointing this out!